gulpfile.js

```diff
@@ -337,6 +338,24 @@ gulp.task('bundle', ['buildnumber'], function () {
   return createBundle(DEFINES).pipe(gulp.dest(BUILD_DIR));
 });
 
+gulp.task('publish', ['generic'], function() {
```

Comment:
Done in the new commit. I decided to make the function asynchronous and chain the streams, so I hope it's better this way!